RomanLoboda/HW3/task03_1.py

```diff
@@ -0,0 +1,29 @@
+input_string=input("Write the line from 'The Zen of Python' file : ").lower()
```

Comment:
good job
look at this more optimal version of code 
input_string = input("Write the line from 'The Zen of Python' file: ").lower()

better_count = input_string.count("better")
never_count = input_string.count("never")
is_count = input_string.count("is")

upper_input_string = input_string.upper()
replaced_text = input_string.replace("i", "&")

print(f"\nThe number of occurrences of the word 'better' is {better_count}")
print(f"The number of occurrences of the word 'never' is {never_count}")
print(f"The number of occurrences of the word 'is' is {is_count}")

print(f"All letters in uppercase: {upper_input_string}")
print(f"The text where 'i' was replaced by '&': {replaced_text}")